modules/util/slice.go

```diff
@@ -12,8 +12,7 @@ import (
 // SliceContainsString sequential searches if string exists in slice.
 func SliceContainsString(slice []string, target string, insensitive ...bool) bool {
 	if len(insensitive) != 0 && insensitive[0] {
-		target = strings.ToLower(target)
-		return slices.ContainsFunc(slice, func(t string) bool { return strings.ToLower(t) == target })
+		return slices.ContainsFunc(slice, func(t string) bool { return strings.EqualFold(t, target) })
```

Comment:
IIRC `SliceContainsString` is only used for ASCII-only cases, for config options or something similar.

Reply:
Sounds dangerous to just assume that for a utility function?

Reply:
Note that `strings.Lower` is also unicode-aware, so code using that vs. `strings.EqualFold` should be strictly compatible.